### PR TITLE
[KOGITO-3024] S2i Archetype build: remove only process file

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-3024

`application.properties` should stay in

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a testcase that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: modules/kogito-s2i-core/added/s2i-core [hunk 1]
```diff
@@ -147,7 +147,7 @@ function build_kogito_app() {
 
         # remove resources generated by the archetype
         log_info "--> Remove archetype generated resources"
-        rm -rf $PROJECT_ARTIFACT_ID/src/main/resources/*
+        rm -rf $PROJECT_ARTIFACT_ID/src/main/resources/test-process.bpmn2
```
Comment (sutaakar) [inline modules/kogito-s2i-core/added/s2i-core]: Where does the test-process.bpmn2 come from?

Comment (radtriste) [reply modules/kogito-s2i-core/added/s2i-core]: generated by the archetype as example project

Comment (radtriste) [reply modules/kogito-s2i-core/added/s2i-core]: https://github.com/kiegroup/kogito-runtimes/tree/master/archetypes/kogito-springboot-archetype/src/main/resources/archetype-resources/src/main/resources

Comment (spolti) [reply modules/kogito-s2i-core/added/s2i-core]: hardcode names is not a good idea.
why instead do this you don't set a few extensions that possible can be in this directory and remove them?

**Review by spolti — CHANGES_REQUESTED** [review]

File: modules/kogito-s2i-core/added/s2i-core (hunk 1 above)
Comment (radtriste) [reply modules/kogito-s2i-core/added/s2i-core]: @spolti updated. let me know

File: modules/kogito-s2i-core/added/s2i-core [hunk 2]
```diff
@@ -147,7 +147,10 @@ function build_kogito_app() {
 
         # remove resources generated by the archetype
         log_info "--> Remove archetype generated resources"
-        rm -rf $PROJECT_ARTIFACT_ID/src/main/resources/*
+        rm -rf $PROJECT_ARTIFACT_ID/src/main/resources/*.bpmn2
```
Comment (spolti) [inline modules/kogito-s2i-core/added/s2i-core]: this also works 
```m -rfv *.{bpm,bpmn2,dmn,drl}```

wdyt:?

Comment (radtriste) [reply modules/kogito-s2i-core/added/s2i-core]: good for me. updated !

**Review by sutaakar — APPROVED** [review]

**Review by spolti — APPROVED** [review]

**Comment by radtriste** [comment]
need https://github.com/kiegroup/kogito-images/pull/212 and new runtimes artifacts deployed with Quarkus 1.7

**Comment by radtriste** [comment]
Jenkins test this

**Comment by radtriste** [comment]
once that one is merged, gonna relaunched the nightly for images&operator

**Pull request merged**